src/templates/earn-page/btc/LatestMints.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Box,
+  BoxProps,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react"
+import { FC } from "react"
+import { BodyMd, H5, LabelSm } from "../../../components"
+import { formatTokenAmount, shortenAddress } from "../../../utils"
+import { getRelativeTime } from "../../../utils/time"
+
+export interface LatestMint {
+  amount: string
+  txHash: string
+  to: string
+  from: string
+  timestamp: string
+}
+
+interface LatestMintProps {
+  latestMints: LatestMint[]
+}
+
+export const LatestMints: FC<LatestMintProps & BoxProps> = ({
+  latestMints,
+  ...restProps
+}) => {
+  return (
+    <Box {...restProps}>
+      <H5 mb="2rem">Latest mints</H5>
+      <TableContainer>
+        <Table variant="simple">
+          <Thead>
+            <Tr>
+              <Th>
+                <LabelSm color="gray.500">Tbtc amount</LabelSm>
+              </Th>
+              <Th>
+                <LabelSm color="gray.500">Wallet Address</LabelSm>
+              </Th>
+              <Th>
+                <LabelSm color="gray.500">TX Hash</LabelSm>
+              </Th>
+              <Th textAlign={"right"}>
+                <LabelSm color="gray.500">Timestamp</LabelSm>
+              </Th>
+            </Tr>
+          </Thead>
+          <Tbody>
+            {latestMints.map((mint, index) => {
+              const backgroundColor = index % 2 ? "transaprent" : "#181D23"
```

Comment:
Probably we can style this row using `_even` and `_odd` prop. Eg `<Tr _odd="#181D23" _odd="transparent" />`. see https://v1.chakra-ui.com/docs/styled-system/features/style-props#pseudo